src/expandableCalendar/WeekCalendar/index.tsx

```diff
@@ -178,14 +183,14 @@ const WeekCalendar = (props: WeekCalendarProps) => {
         }
       }
     }
-  }, [onEndReached]);
+  }, [onEndReached, setDate]);
```

Comment:
I need to change this so there are no dependencies - otherwise there might be an invariant error from flatlist.

Reply:
Done